packages/core/src/schemas/runs.ts

```diff
@@ -78,6 +79,8 @@ export const GetRunSchema = RunSchema.extend({
   output: z.any().optional(),
   /** The tasks from the run */
   tasks: z.array(RunTaskWithSubtasksSchema),
+  /** Any status updates that were published from the run */
+  statuses: z.array(JobRunStatusRecordSchema),
```

Comment:
Let's double check that this isn't going to break any backwards compatibility stuff. Maybe this should be optional or `.default([])`

Reply:
Great point, I've added a default